components/tag/src/tag-list/tag-list.html

```diff
@@ -6,7 +6,10 @@
       *ngIf="_showAllTags"
     ></ng-template>
   </div>
-  <div class="dt-tag-list-more-btn" *ngIf="!_isOneLine && !_showAllTags">
+  <div
+    class="dt-tag-list-more-btn"
+    *ngIf="!_isOneLine && !_showAllTags && _hiddenTagCount > 0"
```

Comment:
NIT: I would create a simple function that evaluates this – keeping templates simple!